assets/js/components/notifications/GatheringDataNotification.js

```diff
@@ -75,35 +74,35 @@ export default function GatheringDataNotification( { id, Notification } ) {
 
 	return (
 		<Notification className="googlesitekit-publisher-win">
-			<NotificationWithSmallSVG
+			<BannerNotification
+				notificationID={ id }
+				type={ TYPES.INFO }
 				title={ gatheringDataTitle }
-				description={
-					<Description
-						text={ sprintf(
-							/* translators: %s: the number of hours the site can be in a gathering data state */
-							_n(
-								'It can take up to %s hour before stats show up for your site. While you’re waiting, connect more services to get more stats.',
-								'It can take up to %s hours before stats show up for your site. While you’re waiting, connect more services to get more stats.',
-								gatheringDataWaitTimeInHours,
-								'google-site-kit'
-							),
-							gatheringDataWaitTimeInHours
-						) }
-					/>
-				}
-				actions={
-					<ActionsCTALinkDismiss
-						id={ id }
-						ctaLabel={ __(
-							'See other services',
-							'google-site-kit'
-						) }
-						ctaLink={ `${ settingsAdminURL }#/connect-more-services` }
-						dismissLabel={ __( 'Maybe later', 'google-site-kit' ) }
-						dismissExpires={ DAY_IN_SECONDS }
-					/>
-				}
-				SmallImageSVG={ GatheringDataIcon }
+				description={ sprintf(
+					/* translators: %s: the number of hours the site can be in a gathering data state */
+					_n(
+						'It can take up to %s hour before stats show up for your site. While you’re waiting, connect more services to get more stats.',
+						'It can take up to %s hours before stats show up for your site. While you’re waiting, connect more services to get more stats.',
+						gatheringDataWaitTimeInHours,
+						'google-site-kit'
+					),
+					gatheringDataWaitTimeInHours
+				) }
+				ctaButton={ {
+					label: __( 'Connect more services', 'google-site-kit' ),
+					href: `${ settingsAdminURL }#/connect-more-services`,
+				} }
+				dismissButton={ {
+					label: __( 'Got it', 'google-site-kit' ),
+				} }
+				dismissOptions={ {
+					expiresInSeconds: DAY_IN_SECONDS,
+				} }
+				svg={ {
+					desktop: GatheringDataSVG,
+					verticalPosition: 'center',
+					hideOnMobile: true,
```

Comment:
I am really trying not to have these inconsistencies and have raised a question for @sigal-teller.